dll/win32/shell32/shellmenu/CStartMenu.cpp

```diff
@@ -168,18 +224,34 @@ class CShellMenuCallback :
         case IDM_PROGRAMS:  csidl = CSIDL_PROGRAMS; break;
         case IDM_FAVORITES: csidl = CSIDL_FAVORITES; break;
         case IDM_DOCUMENTS: csidl = CSIDL_RECENT; break;
+        case IDM_MYDOCUMENTS: csidl = CSIDL_MYDOCUMENTS; break;
+        case IDM_MYPICTURES: csidl = CSIDL_MYPICTURES; break;
         }
 
         if (csidl)
         {
             IShellFolder *psfStartMenu;
+            DWORD dwFlags = SMSET_TOP;
 
             if (csidl == CSIDL_PROGRAMS && m_psfPrograms)
             {
                 psfStartMenu = m_psfPrograms;
             }
             else
             {
+                if (csidl == CSIDL_RECENT)
+                {
+                    BOOL bExpandMyDocuments = FALSE; /* FIXME: Get value from registry */
```

Comment:
Why not setting to "TRUE" by default ?

Reply:
`TRUE` is not default of WinXP and Win2k3.

Reply:
But it could be for ROS. For some other params, i've been asked to use different default values. But choice is yours.